src/it/resources/com/google/checkstyle/test/chapter4formatting/rule451wheretobreak/InputMethodParamPad.java

```diff
@@ -3,224 +3,139 @@
 import java.util.HashMap;
 import java.util.Map;
 
-class InputMethodParamPad
-{
-    class Inner {
-        void testGenerics1
-        () // violation ''(' should be on the previous line.'
+class InputMethodParamPad {
+  class Inner {
+    void testGenerics1
+      () // violation ''(' should be on the previous line.'
         {
-            Comparable
-                <
-                String
-                >
-                c = new String();
```

Comment:
please restore all weird formatting, except for "{" that is not part of modules focus.
we need weird code only for https://checkstyle.org/google_style.html#a4.5.1 modules.

in next PRs you will create InputCorrentMethodParamPad and there will will use auto format to make sure we are not conflicting with formatter.

Reply:
done